### ISSUE TITLE
multitouch index

### ISSUE THREAD

**Issue by KEL-YJ-E** [monkeyswarm/MobMuPlat#29]
Simple but slightly annoying bug: when using the multitouch UI element and touchesByVox, on a computer it starts counting from 1, on your device from 0. So it looks like this:
computer: 1 x y 2 x y 3 x y etc vs device: 0 x y 1 x y 2 x y etc.
I'm on Mac and Android.

**Comment by monkeyswarm** [comment]
Good catch. Looks like iOS app, OSX editor, and Java editor all start touch index at "1", where Android app alone starts it at "0". I will modify the Android app to start at "1" on an upcoming release.
